File: Lab3/socialmediaapp/src/main/java/com/example/socialmediaapp/domain/Post.java
package com.example.socialmediaapp.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@Getter
@Setter
@NoArgsConstructor
public class Post {
    long id;
    String title;
    String content;
    String author;
}

File: Lab9/react-app/src/components/Post/Post.js
import "./Post.css";
const Post = ({ id, title, content, author, clicked }) => {
  return (
    <article className="Post" onClick={clicked}>
      <b>ID:</b> {id} <br />
      <b>Title:</b> {title} <br />
      <b>Author:</b> {author} <br />
      {content}
    </article>
  );
};

export default Post;
